ansible/roles/ooni-explorer-next/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+explorer_tag: '20181112-238cac95'
+explorer_backend_ipv4: 172.28.14.254
+explorer_backend_port: 3000
```

Comment:
It's not a big deal, but BCP is to avoid different variables having same name.
I believe it may trigger some hard-to-debug issues.

Reply:
What is BCP? You mean that this is the same variable name as the `ooni-explorer` playbook?

I guess we can change it, but eventually one of these two playbooks will be dropped and only the next gen explorer will be kept.

Reply:
Best current practice. Yes, I mean that there is same variable name in `ooni-explorer`. I'm sure, it'll not trigger any issues right now, I highlight that mostly for grepability reasons. Please, change names, if that's not too complex.

Also, different services should use different network endpoints. Imagine a case when you have to move a service to another VM in a rush and you face IP conflict as that IP is already present on docker network there.
So, the subnet of backend_ipv4 should be updated to some other [random subnet](https://github.com/ooni/sysadmin/blob/master/scripts/network-endpoint), if you think that it's reasonable to follow that practice.